.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -138,13 +138,15 @@ parameters:
     type: string
     default: ""
 
+  - name: TEST_PLAN_NUM
```

Comment:
Default value should be assigned in test_plan.py
Let's keep empty value in the templates

Reply:
fixed